internal/services/appservice/helpers/function_app_schema.go

```diff
@@ -1374,7 +1375,7 @@ func windowsFunctionAppStackSchema() *pluginsdk.Schema {
 						"site_config.0.application_stack.0.powershell_core_version",
 						"site_config.0.application_stack.0.use_custom_runtime",
 					},
-					Description: "The version of Node to use. Possible values include `12`, `14`, `16` and `18`",
+					Description: "The version of Node to use. Possible values include `12`, `14`, `16`, `18`, `20` and `22`",
```

Comment:
Let's update this to match with the actual expected values and the docs
```suggestion
					Description: "The version of Node to use. Possible values include `~12`, `~14`, `~16`, `~18`, `~20` and `~22`",
```